tests/test_upload_to_gel/test_upload_to_gel.py

```diff
@@ -210,8 +210,6 @@ def _test_deliver_data(self, execute_return, delivered_state):
                          '-o KeepAlive=yes -o BatchMode=yes -o LogLevel=Error -i path/to/id_rsa.pub -p 22" ',
                          '{source} user@gelupload.com:/destination/'.format(source=source))
             mock_execute.assert_any_call(''.join(rsync_cmd))
-
-            assert os.listdir(source) == [self.gel_data_delivery.external_id]
```

Comment:
I originally removed it because I thought it was redundant with the assertion below - `os.listdir` would raise an error if `source/external_id` doesn't exist. I guess one thing the removed `assert` did was make sure there was only one folder in `source` and nothing else. Can reinstate it if we want that feature?

Reply:
Yes please